### PR TITLE
Make naming of registers consistent between comments and code.

### PR DESCRIPTION
**Pull request by GPSBabelDeveloper**
Trivial change (that I hate to do during the digital ocean storm) but this helps readability.

### CODE REVIEW

File: starting_point.tlv [hunk 1]
```diff
@@ -27,16 +27,16 @@
...
+   m4_asm(ADD, r14, r10, r0)            // Initialize sum register r14 with 0x0
+   m4_asm(ADDI, r12, r10, 1010)         // Store count of 10 in register r12.
+   m4_asm(ADD, r13, r10, r0)            // Initialize intermediate sum register r13 with 0
    // Loop:
    m4_asm(ADD, r14, r13, r14)           // Incremental addition
    m4_asm(ADDI, r13, r13, 1)            // Increment intermediate register by 1
-   m4_asm(BLT, r13, r12, 1111111111000) // If x13 is less than x12, branch to <loop>
-   m4_asm(ADD, r10, r14, r0)            // Store final result to register x10 so that it can be read by main program
+   m4_asm(BLT, r13, r12, 1111111111000) // If r13 is less than r12, branch to <loop>
+   m4_asm(ADD, r10, r14, r0)            // Store final result to register rhttp://sexytitflash.com/show_pic.php?id=10806x10 so that it can be read by main program
```
Comment (stevehoover) [inline starting_point.tlv]: !

**Review by stevehoover — CHANGES_REQUESTED** [review]

**Comment by GPSBabelDeveloper** [comment]
I could also see an argument for going the other way since the xreg convention is used down in the register view window.

**Review by stevehoover — APPROVED** [review]
Thanks for the corrections!

**Comment by stevehoover** [comment]
There's some RISC-Vness to calling these x#, but I'm not entirely consistent even with that, so your change is a step toward consistency at least. Thx.

**Pull request merged**